04_numeronym/runnerdave/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[a11y K8s abc]\n")
+	actual := strconv.Quote(buf.String())
+	t.Logf("expected:%s", expected)
+	t.Logf("actual:%s", actual)
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestNumeronyms(t *testing.T) {
+	tables := []struct {
+		x []string
```

Comment:
I think these variable names are a little confusing, I don't know what the x is supposed to stand for. I understand it is the input to the function, but why `x`? Can we update these so it is a little easier to read?
Perhaps, `x -> input` and `n -> expected | output | exp` ?

Will help later on with readability when looking at the line where we check for equality
```go
if !equal(numys, table.expected) {
    ...
}
```

Reply:
good point, thanks